### ISSUE TITLE
Pagination for individual pages

### ISSUE THREAD

**Issue by fabge** [getpelican/pelican#2423]
I am generating a page via the [gallery](https://github.com/getpelican/pelican-plugins/tree/master/gallery) plugin. 
The generated page contains dozens of photos, which is why i want to split them into pages.

I tried using the following syntax from issue #735 but couldn't make it work. I guess this is used for something else than pages.
```
THEME = 'themes/notmyidea'
DIRECT_TEMPLATES = (('index', 'blog', 'tags', 'categories', 'archives'))
PAGINATED_DIRECT_TEMPLATES = (('index', 'blog'))
```

Is there any way to paginate individual pages?

**Comment by avaris** [comment]
No. Those settings are for article collections. Even if you could, `pelican` wouldn't have any information about the photos *inside* your page to paginate. Sounds like this is a job (feature if you will :)) for the `gallery` plugin.
